lib/ui/components/album_horizontal_list_widget.dart

```diff
@@ -9,7 +9,8 @@ import "package:photos/ui/collections/collection_item_widget.dart";
 import "package:photos/ui/common/loading_widget.dart";
 
 class AlbumHorizontalListWidget extends StatefulWidget {
-  final Future<List<CollectionWithThumbnail>> future;
+  final Future<List<CollectionWithThumbnail>> Function() future;
```

Comment:
Should we name this `CollectionsWithThumbnail` instead of `future`?

Reply:
for brevity, changed it to `collectionsFuture`